src/scripts/appclose.ts

```diff
@@ -0,0 +1,11 @@
+((require as unknown) as typeof define)(["playbackManager"], function (playbackManager) {
+    window["AppCloseHelper"] = {
```

Comment:
Why change this from window.AppCloseHelper? The former is the most common way to access window objects.

Reply:
Typescript doesn't know it complains about not knowing it

Reply:
The proper fix would be something like this, according to a quick Google search:

```
declare global {
    interface Window { MyNamespace: any; }
}

window.MyNamespace = window.MyNamespace || {};
```

Reply:
Yeah correct but I don't even know if it'll stay, that type it has and so on, ideally it would be included in the NativeShell typings if it's still a thing

Reply:
It'll need further evaluation. I'll properly fix this in IPC or shortly after it (depending on how I structure those PRs), so I leave it as is right now even though it's kinda a hack